doc/cygnus-ngsi/installation_and_administration_guide/install_from_sources.md

```diff
@@ -25,40 +64,111 @@ Start by cloning the Github repository:
+
+[Top](#top)
 
-Or as a thin Java jar file:
+### <a name="section2.3"></a>Installing `cygnus-ngsi`
 
-    $ cd cygnus-ngsi
-    $ APACHE_MAVEN_HOME/bin/mvn exec:exec package
-    $ cp target/cygnus-<x.y.z>.jar APACHE_FLUME_HOME/plugins.d/cygnus/lib
+    $ cd fiware-cygnus/cygnus-ngsi
+    $ mvn -B -T8 clean compile exec:exec assembly:single
+    $ cp target/cygnus-ngsi-3.6.0-SNAPSHOT-jar-with-dependencies.jar /opt/apache-flume-1.11.0/plugins.d/cygnus/lib/
 
 Finally, please find a `compile.sh` script containing all the commands shown in this section. It must be parameterized with the version of the current branch and the Apache Flume base path.
 
 [Top](#top)
 
-### <a name="section2.3"></a>Known issues
-It may happen while compiling `cygnus-ngsi` the Maven JVM has not enough memory. This can be changed as detailed at the [Maven official documentation](https://cwiki.apache.org/confluence/display/MAVEN/OutOfMemoryError):
+### <a name="section2.4"></a>Setup Cygnus
+
+    $ cp fiware-cygnus/cygnus-common/target/classes/cygnus-flume-ng /opt/apache-flume-1.11.0/bin/
+    $ chmod +x /opt/apache-flume-1.11.0/bin/cygnus-flume-ng
+    $ cp fiware-cygnus/cygnus-common/conf/log4j2.properties.template /opt/apache-flume-1.11.0/conf/log4j2.properties
+    $ cp fiware-cygnus/cygnus-ngsi/conf/name_mappings.conf.template /opt/apache-flume-1.11.0/conf/name_mappings.conf
+    $ mkdir -p /var/log/cygnus
+    $ chown -R cygnus:cygnus /var/log/cygnus
+    
+Clean up
+
+    $ cd fiware-cygnus/cygnus-common
+    $ mvn -B -T8 clean
+    $ cd fiware-cygnus/cygnus-ngsi
+    $ mvn -B -T8 clean>
+    
+[Top](#top)
+
+### <a name="section2.5"></a>Setup Agents
+Setup Cygnus-Common and Cygnus-Ngsi agents
+
+### <a name="section2.6"></a>Running `cygnus-ngsi`
+Please make sure that you have ports 5080 (admin port) and 5050 (default instance port) open.<br/>
 
-    $ export MAVEN_OPTS="-Xmx512m -XX:MaxPermSize=128m"
+    $ /opt/apache-flume-1.11.0/bin/cygnus-flume-ng agent --conf {{PATH-TO-CONF-FOLDER}} -f {{PATH-TO-AGENT-NGSI-CONF}} -n {{AGENT-NAME}} -p 5080 -Dflume.root.logger=INFO,LOGFILE -Duser.timezone=UTC -Dfile.encoding=UTF-8 -Dflume.monitoring.type= -Dflume.monitoring.port=41414
 
 [Top](#top)
 
-## <a name="section3"></a>Installing dependencies
-These are the packages you will need to install under `APACHE_FLUME_HOME/plugins.d/cygnus/libext/` **if you did not included them in the cygnus-common jar**:
-
-| Cygnus dependencies | Version | Required by / comments |
-|---|---|---|
-| mockito-all | 1.9.5 | Unit tests |
-| junit | 4.11 | Unit tests |
-| httpcore | 4.3.1 | Overwrites the one bundled in Apache Flume |
-| log4j | 1.2.17 | Logging |
-| slf4j-simple | 1.7.21 | Logging |
-| cygnus-common | latest | |
+## <a name="section3"></a>SSL Support
+Apache Flume has SSL support, <a href='https://flume.apache.org/FlumeUserGuide.html#ssl-tls-support'>mentioned here</a>
```

Comment:
Maybe just say that SSL is an option

Reply:
In addition, I'd suggest to use native markdown links instead HTML `<a>` tag, I mean:

```suggestion
Apache Flume has SSL support, [mentioned here](https://flume.apache.org/FlumeUserGuide.html#ssl-tls-support)
```

If you have more links in the PR, please fix them the same way.